### DISCUSSION TITLE
Adding the http version (e.g. HTTP/1.1) to the request doesn't work

### DISCUSSION THREAD

**Discussion by ppenguin** [rest-nvim/rest.nvim#488] [Q&A]
According to [the spec](https://www.jetbrains.com/help/idea/exploring-http-syntax.html#provide-request-body) I should be able to do
```
POST {{url}}/some/path HTTP/1.1
```
but the protocol is interpreted as part of the query and leads to `404`.

How to specify the needed protocol?

**Comment by boltlessengineer** [comment]
Thank you for reporting. Seems to be a tree-sitter parser bug.
Transferring this discussion to rest-nvim/tree-sitter-http#44
